solutions/banking/variables.tf

```diff
@@ -96,11 +96,23 @@ variable "watson_machine_learning_instance_resource_name" {
 }
 
 variable "signing_key" {
-  description = "Signing GPG key."
+  description = "Signing GPG key. If it is not provided by the user, then automation will create one."
```

Comment:
If this is optional, should this take a default?

Reply:
I don't think we can feasibly provide a "default" GPG key - either the user provides one, or the system generates one at apply time.